src/components/SubscriptionForm.js

```diff
@@ -1,29 +1,45 @@
-import React from 'react';
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux'
+import { setSubscribed } from '../actions'
 
-class SubscriptionForm extends React.Component {
-  constructor(){
-    super();
-    this.state = { value: '', submitted: false }
-    this.handleChange = this.handleChange.bind(this);
-    this.handleSubmit = this.handleSubmit.bind(this);
-  }
-
-  handleChange(e) {
-    this.setState({value: e.target.value});
-  }
-
-  handleSubmit(e) {
-    e.preventDefault();
-  }
-
-  render() {
+const SubscriptionForm = ({handleSubmit, subscribed}) => {
+  if(subscribed){
```

Comment:
Perhaps the ThankYouMessage could be made more generic (like a BannerMessage component that takes in a message to be displayed as a prop) - this might be a useful component in the future.

Also, great idea on renaming to SubscriptionView. It's a lot more descriptive than what I have now.

@thegame8714 Thank you for reviewing my work - really appreciate it!